app/services/notify_extra_mobile_data_account_holder_service.rb

```diff
@@ -0,0 +1,58 @@
+class NotifyExtraMobileDataAccountHolderService
+  class NotifySmsError < StandardError; end
+
+  # def self.call(extra_mobile_data_request)
+  #   new(extra_mobile_data_request).call
+  # end
+
+  # def initialize(extra_mobile_data_request)
+  #   @extra_mobile_data_request = extra_mobile_data_request
+  # end
```

Comment:
I think it'd be better to get rid of the commented-out code...

Reply:
Good spot! I agree on the renaming, perhaps we could look at a bit of refactoring when this gets moved into background jobs along the lines you mention above so we can merge this in?

Reply:
👍